### PR TITLE
50:bottom-nav

### PR DESCRIPTION
**Pull request by tolia-ogurt**

### CODE REVIEW

File: app/src/main/java/com/example/trempel/BottomNavExtension.kt [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+import androidx.core.util.forEach
+import androidx.fragment.app.FragmentManager
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.navigation.NavController
+import androidx.navigation.fragment.NavHostFragment
+import com.example.trempel.R
+import com.google.android.material.bottomnavigation.BottomNavigationView
+
+fun BottomNavigationView.setupWithNavController(
```
Comment (DaryaErofeeva) [inline app/src/main/java/com/example/trempel/BottomNavExtension.kt]: can we change it to the `internal` function?

File: app/src/main/java/com/example/trempel/ui/bag/BagFragment.kt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.example.trempel.ui.bag
+
+import androidx.fragment.app.Fragment
+import com.example.trempel.R
+
+class BagFragment : Fragment(R.layout.bag_fragment) {
```
Comment (DaryaErofeeva) [inline app/src/main/java/com/example/trempel/ui/bag/BagFragment.kt]: `internal`?

File: app/src/main/java/com/example/trempel/ui/favourites/FavouritesFragment.kt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+package com.example.trempel.ui.favourites
+
+import androidx.fragment.app.Fragment
+import com.example.trempel.R
+
+class FavouritesFragment: Fragment(R.layout.favourites_fragment) {
```
Comment (DaryaErofeeva) [inline app/src/main/java/com/example/trempel/ui/favourites/FavouritesFragment.kt]: `internal`?

File: app/src/main/java/com/example/trempel/ui/profile/ProfileFragment.kt [hunk 4]
```diff
@@ -0,0 +1,7 @@
+package com.example.trempel.ui.profile
+
+import androidx.fragment.app.Fragment
+import com.example.trempel.R
+
+class ProfileFragment : Fragment(R.layout.profile_fragment) {
```
Comment (DaryaErofeeva) [inline app/src/main/java/com/example/trempel/ui/profile/ProfileFragment.kt]: `internal`?

File: app/src/main/res/drawable/bottom_nav_colors.xml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:color="@color/black" android:state_checked="true" />
+
```
Comment (DaryaErofeeva) [inline app/src/main/res/drawable/bottom_nav_colors.xml]: please, remove redundant empty line

File: app/src/main/java/com/example/trempel/BottomNavExtension.kt (hunk 1 above)
Comment (tolia-ogurt) [reply app/src/main/java/com/example/trempel/BottomNavExtension.kt]: done

File: app/src/main/java/com/example/trempel/ui/bag/BagFragment.kt (hunk 2 above)
Comment (tolia-ogurt) [reply app/src/main/java/com/example/trempel/ui/bag/BagFragment.kt]: done

File: app/src/main/java/com/example/trempel/ui/favourites/FavouritesFragment.kt (hunk 3 above)
Comment (tolia-ogurt) [reply app/src/main/java/com/example/trempel/ui/favourites/FavouritesFragment.kt]: done

File: app/src/main/java/com/example/trempel/ui/profile/ProfileFragment.kt (hunk 4 above)
Comment (tolia-ogurt) [reply app/src/main/java/com/example/trempel/ui/profile/ProfileFragment.kt]: done

File: app/src/main/res/drawable/bottom_nav_colors.xml (hunk 5 above)
Comment (tolia-ogurt) [reply app/src/main/res/drawable/bottom_nav_colors.xml]: done

**Review by DaryaErofeeva — APPROVED** [review]

**Pull request merged**